can/SConscript

```diff
@@ -1,20 +1,14 @@
 Import('env', 'envCython', 'cereal', 'common')
 
 import os
-from opendbc.can.process_dbc import process
 
-dbcs = []
-for x in sorted(os.listdir('../')):
-  if x.endswith(".dbc"):
-    def compile_dbc(target, source, env):
-      process(source[0].path, target[0].path)
-    in_fn = [os.path.join('../', x), 'dbc_template.cc']
-    out_fn = os.path.join('dbc_out', x.replace(".dbc", ".cc"))
-    dbc = env.Command(out_fn, in_fn, compile_dbc)
-    env.Depends(dbc, ["dbc.py", "process_dbc.py"])
-    dbcs.append(dbc)
-
-libdbc = env.SharedLibrary('libdbc', ["dbc.cc", "parser.cc", "packer.cc", "common.cc"]+dbcs, LIBS=[common, "capnp", "kj", "zmq"])
+envDBC = env.Clone()
+dbc_file_path = '-DDBC_FILE_PATH=\'"%s"\'' % (envDBC.Dir("..").abspath)
+if isinstance(envDBC['CXXFLAGS'], list):
+  envDBC['CXXFLAGS'] += [dbc_file_path]
+else:
+  envDBC['CXXFLAGS'] += ' ' + dbc_file_path
```

Comment:
why isn't this always a list? we assume it is in other SConscripts

Reply:
ah right, it is. I could have sworn I printed it and it was a string on my PC but list in CI